iothub/device/src/Transport/HttpTransportHandler.cs

```diff
@@ -427,6 +421,29 @@ private static TimeSpan GetInvokeDeviceMethodOperationTimeout(MethodInvokeReques
             return customHeaders;
         }
 
+        private static object ConvertToMessageSystemProperty(string messagePropertyName, IEnumerable<string> headerValue)
+        {
+            string propertyValue = headerValue?.First();
+
+            if (messagePropertyName.Equals(MessageSystemPropertyNames.EnqueuedTime, StringComparison.Ordinal)
```

Comment:
Just double checking that case-sensitive comparison is intentional.

Reply:
yes, the case sensitive comparison is intentional